PM2-README.md

```
@@ -0,0 +1,329 @@
+pm2 logs azure-devops-bot --lines 100  # Last 100 log lines
+pm2 describe azure-devops-bot           # Detailed process info
+pm2 reset azure-devops-bot              # Reset restart counter
+```
+
+## 📊 Monitoring & Logs
+
+### Log Files
+
+PM2 automatically creates and manages log files:
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)